stack/python/stack.py

```diff
@@ -0,0 +1,52 @@
+class Stack:
+    """
+    Class for operations related to stack.
+    """
+    def __init__(self):
+        """
+        Constructor.
+        """
+        self.stack = []    # Initialise empty list
+
+    def push(self, dataval):
+        """
+        Method to push items in stack.
+
+        dataval: Value that user needs to push.
+        """
+        self.stack.append(dataval)    # Add dataval at the end of the list.
+
+    def pop(self):
+        """
+        Method for removing the last element from the stack.
+
+        Returns
+        -------
+        element that is popped.
+        """
+        if len(self.stack) <= 0:
+            return ("No element in the Stack")    # Raise an error is if stack is empty.
+        else:
+            return self.stack.pop()
+
+    def top(self):
+        return self.stack[len(self.stack)-1]    # Show the last element of the list.
```

Comment:
E501 line too long (84 > 80 characters)

*Origin: PycodestyleBear (E501), Section: `all.autopep8`.*

Reply:
E501 is ignored in some core python libraries too! Do you want me to fix it? Even autopep8 ignores that xD